helpers/utilityfunctions.py

```diff
@@ -654,3 +654,9 @@ def previous(self):
 
     def current(self):
         return self.items[self.index]
+
+# people with `, *, etc, in their usernames break a ton of formatting
+def escape_formatting_chars(s):
+    for c in ["*", "_", "`", "~", ">"]: # frankly im not sure if > has any effect in embeds but i'll leave it here
```

Comment:
Technically surrounding a link with < and > will stop it from expanding, but this only applies to things discord detects as links, so aside from very rear cases like a user having <www.example.com> in their usernames it's not going to be absorbed